dataMaker.js

```diff
@@ -0,0 +1,122 @@
+const featTraits = ['Great Location', 'Super host', 'Sparkling Clean', 'Breakfast', 'Self check-in', 'Great check-in experience'];
+function dataMaker(writer) {
+  let i = 10000001;
+  function write() {
+    let ok = true;
+    do {
+      const data = [
+        randomNumber[Math.floor(Math.random() * randomNumber.length)],
+        randomNumber[Math.floor(Math.random() * randomNumber.length)],
+        randomNumber[Math.floor(Math.random() * randomNumber.length)],
+        randomNumber[Math.floor(Math.random() * randomNumber.length)],
+        fakeSentence,
+        homeType[Math.floor(Math.random() * homeType.length)],
+        hostName[Math.floor(Math.random() * hostName.length)],
+        count += 1,
+        fakeSentence,
+        neighborhoods[Math.floor(Math.random() * neighborhoods.length)],
+        featTraits[Math.floor(Math.random() * featTraits.length)],
+        featTraits[Math.floor(Math.random() * featTraits.length)],
+        featTraits[Math.floor(Math.random() * featTraits.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+        randomBoolean[Math.floor(Math.random() * randomBoolean.length)],
+      ];
```

Comment:
I like the refactor to eliminate faker data generation. Runs much faster I'm sure. However, lines 78-103 are pretty dirty. consider running a for loop and pushing values into the array, or something kind of like that. Maybe a Math.random helper function. Also, you don't need all of that logic in the brackets.

Could use something like ```randomBoolean[randomNumFunc()]```